src/commands/google.ts

```diff
@@ -0,0 +1,77 @@
+import {CommandType} from "../types";
+import {logger} from "@/config";
+import process from "process";
+
+import {
+  CacheType,
+  SlashCommandBuilder,
+  ChatInputCommandInteraction,
+  EmbedBuilder,
+  Colors,
+} from "discord.js";
+
+const googleModule: CommandType = {
+  data: new SlashCommandBuilder()
+    .setName("google")
+    .setDescription("Google something and return the top 10 results.")
+    .addStringOption((option) =>
+      option
+        .setName("query")
+        .setDescription("What you want to google.")
+        .setRequired(true)
+    )
+    .addUserOption((option) =>
+      option
+        .setName("to")
+        .setDescription("The user you want to direct this to.")
+    ),
+  execute: async (interaction: ChatInputCommandInteraction<CacheType>) => {
+    const query = interaction.options.getString("query");
+    const toWhom = interaction.options.getUser("to");
+    const embed = new EmbedBuilder()
+      .setColor(Colors.Blue)
+      .setAuthor({
+        name: `Requested by ${interaction.user.displayName}`,
+        iconURL: `${interaction.user?.avatarURL()}`,
+      })
+      .setTimestamp();
+
+    const search_key = process.env.GOOGLE_SEARCH_API_KEY;
+    const search_id = process.env.GOOGLE_SEARCH_ID;
+    const url = `https://www.googleapis.com/customsearch/v1?key=${search_key}&cx=${search_id}&q=${query}`;
+
+    let links: string[] = [];
+
+    await fetch(url, {
+      method: "GET",
+      headers: {Accept: "*/*"},
+    })
+      .then((response) => {
+        if (!response.ok) {
+          logger.info("No response.");
+          interaction.reply({
+            content: "Unable to return any queries.",
+            ephemeral: true,
+          });
+        }
+
+        return response.json();
+      })
+      .then((data) => {
+        let i: number = 0;
+        data.items.forEach((element: any) => {
```

Comment:
Would be nice to create an interface for this. Can exist only in this file.

Any should generally be avoided because its usually a cop-out of explicitly stating the type you expect.